plutarch-test/plutarch-test.cabal

```diff
@@ -133,6 +134,7 @@ executable plutarch-test
     other-modules:
       Plutarch.FieldSpec
       Plutarch.MonadicSpec
+      Plutarch.TryFromSpec
```

Comment:
I don't think we use any GHC 9 features here

Reply:
I added it because something was broken that sid fixed and I left it like that to be able to demonstrate the `HasField` instance